src/kaltura-player.js

```diff
@@ -78,7 +95,7 @@ class KalturaPlayer extends FakeEventTarget {
     const playerConfig = Utils.Object.copyDeep(mediaConfig);
     Utils.Object.mergeDeep(playerConfig.sources, this._localPlayer.config.sources);
     Utils.Object.mergeDeep(playerConfig.session, this._localPlayer.config.session);
-    Object.keys(this._localPlayer.config.plugins).forEach(name => {
+    Object.keys(this._localPlayer.config.plugins || {}).forEach(name => {
```

Comment:
Why not saving the `options` on the `this` and access it by `this._options.plugins`?

Reply:
@yairans ?

Reply:
this code is outdated @dan-ziv